Jellyfin.Api/Controllers/MusicGenresController.cs

```diff
@@ -109,15 +109,15 @@ public class MusicGenresController : BaseJellyfinApiController
                 EnableTotalRecordCount = enableTotalRecordCount
             };
 
-            if (!string.IsNullOrWhiteSpace(parentId))
+            if (parentId.HasValue)
             {
                 if (parentItem is Folder)
                 {
-                    query.AncestorIds = new[] { new Guid(parentId) };
+                    query.AncestorIds = new[] { parentId.Value };
```

Comment:
Is `.Value` the Guid here? `Guid?` -> `Guid`?

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.nullable-1.value?view=net-5.0
`parentId` is now a nullable `Guid`, so just pass the value.

Reply:
So, kinda similar to the nullable stuff in Rust then.